### PR TITLE
Add commentary on how to get these cmdlets if you don't have them in …

### LINKED ISSUE #1045: Add section to explain how to get these cmdlets if they are missing from your machine
I spent a decent amount of time trying to find out how to add these cmdlets to my powershell environment.  I think all of these are in the Windows Feature RSAT-Clustering-Powershell.  I'll propose a PR which indicates that.  It would be very helpful if the rest of the documents here also explained how to get the cmdlets if they're missing in your powershell environment.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0a3e12c0-7173-c914-7807-4baf68994331
* Version Independent ID: 7a47799e-a3ca-e16d-27d8-e288a129632e
* Content: [FailoverClusters](https://docs.microsoft.com/en-us/powershell/module/failoverclusters/?view=win10-ps)
* Content Source: [docset/windows/failoverclusters/failoverclusters.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/failoverclusters/failoverclusters.md)
* Product: **w10**
* Technology: **powershell-windows**
* GitHub Login: @coreyp-at-msft
* Microsoft Alias: **coreyp**

### PR DESCRIPTION
**Pull request by MarkIannucci**
…your powershell environment.

closes #1045 .

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR contributed by the community. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR contributed by the community. Thanks.

**Comment by dariomws306** [comment]
Maybe we can try with something like:

**Comment by dariomws306** [comment]
Maybe we can try with something like:

If you don't have the Failover Clusters module installed on your machine, you need to download the correct Remote Server Administration Tools (RSAT) package for your OS. If you're running windows 7, you will also need to run the Import-Module FailoverClusters command from an elevated PowerShell prompt. For more information about Remote Server Administration Tools (RSAT) please go to [Remote Server Administration Tools (RSAT) for Windows operating systems](https://support.microsoft.com/help/2693643/remote-server-administration-tools-rsat-for-windows-operating-systems).

Maybe we can find a better article for RSAT, I also found this:
https://docs.microsoft.com/en-us/windows-server/remote/remote-server-administration-tools#install-uninstall-and-turn-offon-rsat-tools

**Comment by dariomws306** [comment]
Or we can add both links... Support KB it's not up to date with the October 2018 update. But you can find that when you try to download RSAT for Win 10.

Also I think it would be idea to add this text to all or almost all cmdlets list.

**Comment by get-itips** [comment]
I agree with Dario that the explanation can be extended a little more to cover more Operating Systems and have a link to explain RSAT. Also, thanks for the contribution @MarkIannucci

File: docset/windows/failoverclusters/failoverclusters.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ ms.assetid: 1149AB0F-CE0C-4F56-8352-15D29814411F
 
 # FailoverClusters Module
 ## Description
-This reference provides cmdlet descriptions and syntax for all failover cluster-specific cmdlets. It lists the cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.
+This reference provides cmdlet descriptions and syntax for all failover cluster-specific cmdlets. It lists the cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.  If these cmdlets aren't available in your powershell session, you may need to add the `Failover Cluster Module for Windows PowerShell` Feature.  To do that, use this cmd: `Add-WindowsFeature RSAT-Clustering-PowerShell`
```
Comment (JohanFreelancer9) [inline docset/windows/failoverclusters/failoverclusters.md]:
```suggestion
This reference provides cmdlet descriptions and syntax for all failover cluster-specific cmdlets. It lists the cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.  If these cmdlets aren't available in your PowerShell session, you may need to add the `Failover Cluster Module for Windows PowerShell` Feature.  To do this, use this cmd: `Add-WindowsFeature RSAT-Clustering-PowerShell`.
```

Comment (nenonix) [inline docset/windows/failoverclusters/failoverclusters.md]:
```suggestion
This reference provides cmdlet descriptions and syntax for all failover cluster-specific cmdlets. It lists the cmdlets in alphabetical order based on the verb at the beginning of the cmdlet.  If these cmdlets aren't available in your PowerShell session, you may need to add the "FailoverClusters" module for Windows PowerShell using the `Add-WindowsFeature RSAT-Clustering-PowerShell` cmdlet.
```

**Comment by tseward** [comment]
The updated PR looks good.

**Comment by e0i** [comment]
@AndreaBarr This PR that has been contributed by the community and passed copy and technical review can be merged. Thanks.

**Pull request merged**